contracts/meta-asset-token/MetaAssetToken.sol

```diff
@@ -31,7 +31,7 @@ contract MetaAssetToken is ERC20Permit, Ownable {
     /**
      * @dev Emitted when transfer  Manager config is changed.
      */
-    event TransferWithPermit(address _from, address _to, uint256 _amount);
+    event TransferWithPermit(address indexed _from, address indexed _to, uint256 _amount);
```

Comment:
pls fix this vulnerability in MetaAssetToken contract too for future deployment and also include all the dependent contracts inheriting it in this upgrade with explanation that this is for consistency because their bytecode changed

Reply:
updated in this [commit](https://github.com/DistributedCollective/mynt/pull/17/commits/0b495d7b734a64db865d48d07a012ef1cefba6b4)